api/src/main/java/openconsensus/trace/data/Event.java

```diff
@@ -30,54 +30,54 @@
  */
 @Immutable
 @AutoValue
-public abstract class Annotation {
+public abstract class Event {
   private static final Map<String, AttributeValue> EMPTY_ATTRIBUTES =
       Collections.unmodifiableMap(Collections.<String, AttributeValue>emptyMap());
 
   /**
-   * Returns a new {@code Annotation} with the given description.
+   * Returns a new {@code Event} with the given name.
    *
-   * @param description the text description of the {@code Annotation}.
-   * @return a new {@code Annotation} with the given description.
-   * @throws NullPointerException if {@code description} is {@code null}.
+   * @param name the text name of the {@code Event}.
+   * @return a new {@code Event} with the given name.
+   * @throws NullPointerException if {@code name} is {@code null}.
    * @since 0.1.0
    */
-  public static Annotation fromDescription(String description) {
-    return new AutoValue_Annotation(description, EMPTY_ATTRIBUTES);
+  public static Event fromDescription(String name) {
+    return new AutoValue_Event(name, EMPTY_ATTRIBUTES);
   }
 
   /**
-   * Returns a new {@code Annotation} with the given description and set of attributes.
+   * Returns a new {@code Event} with the given name and set of attributes.
    *
-   * @param description the text description of the {@code Annotation}.
-   * @param attributes the attributes of the {@code Annotation}.
-   * @return a new {@code Annotation} with the given description and set of attributes.
-   * @throws NullPointerException if {@code description} or {@code attributes} are {@code null}.
+   * @param name the text name of the {@code Event}.
+   * @param attributes the attributes of the {@code Event}.
+   * @return a new {@code Event} with the given name and set of attributes.
+   * @throws NullPointerException if {@code name} or {@code attributes} are {@code null}.
    * @since 0.1.0
    */
-  public static Annotation fromDescriptionAndAttributes(
-      String description, Map<String, AttributeValue> attributes) {
-    return new AutoValue_Annotation(
-        description,
+  public static Event fromDescriptionAndAttributes(
+      String name, Map<String, AttributeValue> attributes) {
+    return new AutoValue_Event(
+        name,
         Collections.unmodifiableMap(
             new HashMap<String, AttributeValue>(Utils.checkNotNull(attributes, "attributes"))));
   }
 
   /**
-   * Return the description of the {@code Annotation}.
+   * Return the description of the {@code Event}.
    *
-   * @return the description of the {@code Annotation}.
+   * @return the description of the {@code Event}.
    * @since 0.1.0
    */
   public abstract String getDescription();
 
   /**
-   * Return the attributes of the {@code Annotation}.
+   * Return the attributes of the {@code Event}.
    *
-   * @return the attributes of the {@code Annotation}.
+   * @return the attributes of the {@code Event}.
    * @since 0.1.0
    */
   public abstract Map<String, AttributeValue> getAttributes();
```

Comment:
only name and attributes are the fields of the event? what about the timestamp?

Reply:
You only get/set name and attribute. Time is automatically stamped on `addEvent`. `SpanData` gives an Event with timestamp (different data type).

Reply:
Given that we are removing the ability to set timestamps for span `start` and `finish` events, it makes sense to me that we do not expose it here either.

@yurishkuro does this sound acceptable to you?